Dockerfile

```diff
@@ -1,11 +1,19 @@
-FROM node:18
+FROM node:18 as Builder
+
+WORKDIR /action
+
+RUN npm i -g @vercel/ncc
```

Comment:
Can we pin the version here?

Reply:
Let's add it to the build dev dependencies instead, so dependabot can keep it updated. What do you think?

Reply:
Perfect.